src/components/Loadable/Loadable.driver.ts

```diff
@@ -0,0 +1,34 @@
+import { ComponentFactory } from 'wix-ui-test-utils/driver-factory';
+import { BaseDriver } from 'wix-ui-test-utils/driver-factory';
+
+export interface LoadableDriver extends BaseDriver {
+  isLoaded: () => boolean;
+  toggleShouldLoadComponents: () => {}
+}
+
+export const loadableDriverFactory = ({
+  childFactory,
+  childSelector,
+}: {
+  childFactory: (arg: ComponentFactory) => BaseDriver;
+  childSelector: (element: Element) => Element;
+}) => ({
+  element,
+  eventTrigger,
+  wrapper,
```

Comment:
we don't use wrapper and component anymore

Reply:
But according to wix-ui-test-utils typings, only `component` is optional.
https://github.com/wix/wix-ui/blob/master/packages/wix-ui-test-utils/src/driver-factory/createDriverFactory.tsx#L16

Reply:
Right, but we don’t use them anymore. Test utils is a bit of a mess of old and new approach, so I think it’s better to remove the wrapper